configs/_base_/models/stylegan/stylegan2_base.py

```diff
@@ -16,7 +16,7 @@
 
 model = dict(
     type='StyleGAN2',
-    data_preprocessor=dict(type='GANDataPreprocessor'),
+    data_preprocessor=dict(type='mmgen.GANDataPreprocessor'),
```

Comment:
Maybe we can use `register_all_modules` in `inference.py` rather than add `mmgen` prefix in our config.